docs/development.md

```diff
@@ -246,10 +242,6 @@ docker compose run -p 8484:8484 datastore
 ```
 If the above command does not work, you may want to try this instead:
 
-```sh
```

Comment:
Some comments for this particular panel section:
- "If the above command does not work, you may want to try this instead:" -> this line should be removed as well
- Let's standardise the language used, so "We have a Docker compose definition to run dependent services..." can be changed to something like "Docker Compose v2 is used to run dependent services..." (i.e. follow the change you made in line 66)